yb-voyager/src/tgtdb/event.go

```diff
@@ -21,11 +21,11 @@ import (
 )
 
 type Event struct {
-	Op         string            `json:"op"`
-	SchemaName string            `json:"schema_name"`
-	TableName  string            `json:"table_name"`
-	Key        map[string]string `json:"key"`
-	Fields     map[string]string `json:"fields"`
+	Op         string                 `json:"op"`
+	SchemaName string                 `json:"schema_name"`
+	TableName  string                 `json:"table_name"`
+	Key        map[string]interface{} `json:"key"`
+	Fields     map[string]interface{} `json:"fields"` 
```

Comment:
Strictly no!! Keep the values coming from debezium as `string`. Only the value converter functions should interpret the values. We don't want golang type semantics, json type semantics to come into play. e.g. consider a high precision float number or decimal number.

Reply:
@makalaaneesh, can you please make that change again?